Makefile

```diff
@@ -1,3 +1,15 @@
+# If JOBS environment variable not defined, default to the number of CPUs available
+ifeq ($(JOBS),)
+JOBS:=1
+OS:=$(shell uname -s)
+ifeq ($(OS),Linux)
+  JOBS:=$(shell grep -c ^processor /proc/cpuinfo)
```

Comment:
Well, this clause is already checking for `Linux`, so I feel confident that `/proc/cpuinfo` will exist.  To be honest, I just copy/pasted this from somewhere else.  I don't feel strongly either way, just as long as it works on Travis :-)